Assignment Specific Content

G-Henderson opened the pull request:
Added reflections.

deserted-operative committed:
Create .mailmap

Created a .mailmap file so that in the Git log, the various accounts and email addresses with the team's commits associate to the names of the correct individuals.

deserted-operative committed:
Create Assignment Report & Reflections.md

Added the template for the assignment report and teammate reflections.

Hasan24642 committed:
Update Assignment Report & Reflections.md

Hasan24642 committed:
Update Assignment Report & Reflections.md

parlatellioglu committed:
Update Assignment Report & Reflections.md

deserted-operative committed:
Update Assignment Report & Reflections.md

Added a draft for the 200 word section on what we accomplished in this prototyping project.

deserted-operative commented:
I've added a draft for the 200 word section on _what_ we accomplished.
Any and all changes welcome.

G-Henderson commented:
Will have a look

parlatellioglu committed:
Update Assignment Report & Reflections.md

G-Henderson committed:
Update Assignment Report & Reflections.md

Added my reflections

deserted-operative committed:
Moved assignment report & reflections

Moved the assignment report and reflections document out from the wrong directory to make it easier to discover.

deserted-operative committed:
Update Assignment Report & Reflections.md

Added draft for 800 word "how we did it" section.

Note that there's about 50 words to still play with, within the limit.

deserted-operative committed:
Update Assignment Report & Reflections.md

Fixed a typo

deserted-operative commented:
The draft is only about 750 of the 800 words, and will almost certainly benefit from editing/cutting down some of what's there.
Any more detail to be added about the SSH App would also be great.

I propose that once we have the language sorted, we _then_ add images?

Let me know :)

deserted-operative committed:
Update Assignment Report & Reflections.md

Added my teammate reflections.

G-Henderson committed:
Update Assignment Report & Reflections.md

Made some changes to the write up

deserted-operative committed:
Create Issues Tracking - Milestone 2.png

Added an image for the 800 word "how we did it" write up.

G-Henderson committed:
Added images

Added kanban image and rulesets image

deserted-operative committed:
Added images for 800 word write up

Added two images for the 800 "how we did it" write up.

deserted-operative committed:
Merge branch 'assignment-specific-content' of https://github.com/SEPP-SSH/Shared-Groceries into assignment-specific-content

G-Henderson committed:
Create Successful Action.png

deserted-operative committed:
Update Assignment Report & Reflections.md

Added stubs for images to be added to.

deserted-operative committed:
Merge branch 'assignment-specific-content' of https://github.com/SEPP-SSH/Shared-Groceries into assignment-specific-content

G-Henderson committed:
Update Assignment Report & Reflections.md

Added the image paths
